adoc/deployment-vmware.adoc

```diff
@@ -275,19 +275,32 @@ terraform plan
 terraform apply
 ----
 
-==== Setup by Hand
+==== Setup with {ay}
 For each VM deployment, follow the {ay} installation method used for deployment on
 bare metal machines as described in <<deployment_bare_metal>>.
 
+==== Setup by Hand
+Deploy the template to your created VMs. After that, boot into the node and configure
+the OS as needed.
+
+.Regenerating Machine ID
 [IMPORTANT]
 ====
-Make sure to give each VM in VMware a clear name that shows its purpose in the cluster, for example
+During the template preparation you will have removed the machine ID from the template image.
```

Comment:
Please move this para about regeneration of machine-id into "Deploying VMs from the Template" section and introduce the para by something like:
```In case you are using template but NOT terraform for deployment please do:```